examples/configurables/configuration.toml

```diff
@@ -0,0 +1,8 @@
+hostName = "192.0.1.2"
```

Comment:
I'm not sure if we support toml files in BBEs. Looping in @lafernando @praneesha 

If it is not supported, it may be OK to have the file for someone who needs to look it up on GitHub, but for the example to make sense, we'll have to add this maybe to the .out file like in https://ballerina.io/swan-lake/learn/by-example/config-api.html, right?

Reply:
Yes, as in, we can of course just have that file in the GH repo, but you don't see the contents of that in the BBE web page. We either have to show the contents of it in the .out file, or else, maybe we can put a link to the file in GH in the .out file, so people can navigate to it. Now, when we run this, don't we have to provide where the configuration file is at, like we earlier did in config api with "--b7a.config.file"?

Reply:
And we should also mention how the configuration values can be given, is it only the TOML file, will it read from the env variables too?

Reply:
This feature is in the initial stage of the implementation. Currently, we only support the configuration TOML file in the running directory and with specific name `configuration.toml`  for slp6. We'll be supporting the other possibilities in future. 
[#27127 ](https://github.com/ballerina-platform/ballerina-lang/issues/27127)

I updated the toml content in the `.out` file. Please review.

Reply:
@MaryamZi @lafernando @praneesha

Reply:
Ack.